app/styles/stepper.module.css

```diff
@@ -27,14 +27,15 @@
 }
 
 .label {
-  color: var(--color-black);
-  font-weight: 700;
-  font-size: 1.25rem;
-  margin: 0 0 2px 2px;
+  color: var(--neutral-neutral-800, #1F1F1F);
+  font-size: 16px;
```

Comment:
In some places we use `rem` and in some places `px`.
Can we make it consistent?

Reply:
+!

Reply:
Done with changes can you please check latest commit